### ISSUE TITLE
Making toolbar Responsive

### ISSUE THREAD

**Issue by keshav234156** [publiclab/PublicLab.Editor#496]
Toolbar is cropped in Mobile view and small Screen View
![Screenshot from 2020-02-22 21-59-43](https://user-images.githubusercontent.com/45951376/81216518-a6200880-8ff8-11ea-99d6-f7220c725782.png)

**Comment by imrishabh18** [comment]
Can I take up this issue ?

**Comment by keshav234156** [comment]
@imrishabh18 Yes you,can take up this issue. This is a First timer only issue(FTO) https://github.com/publiclab/plots2/issues/7067

**Comment by dharmelolar** [comment]
hello @keshav234156 please can i claim this issue, i see that it has not been worked on.

**Comment by keshav234156** [comment]
@dharmelolar yes, you can take up this issue.
